README.md

```diff
@@ -1,7 +1,7 @@
 
 # SR-IOV Network device plugin for Kubernetes
 
-[![Travis CI](https://travis-ci.org/intel/sriov-network-device-plugin.svg?branch=master)](https://travis-ci.org/intel/sriov-network-device-plugin/builds) [![Go Report Card](https://goreportcard.com/badge/github.com/intel/sriov-network-device-plugin)](https://goreportcard.com/report/github.com/intel/sriov-network-device-plugin)
+[![Travis CI](https://travis-ci.org/intel/sriov-network-device-plugin.svg?branch=master)](https://travis-ci.org/intel/sriov-network-device-plugin/builds) [![Go Report Card](https://goreportcard.com/badge/github.com/k8snetworkplumbingwg/sriov-network-device-plugin)](https://goreportcard.com/report/github.com/k8snetworkplumbingwg/sriov-network-device-plugin)
```

Comment:
why you didn't update /travis-ci.org/intel to /travis-ci.org/k8snetworkplumbingwg? I see https://travis-ci.org/github/k8snetworkplumbingwg exist

Reply:
> why you didn't update /travis-ci.org/intel to /travis-ci.org/k8snetworkplumbingwg? I see https://travis-ci.org/github/k8snetworkplumbingwg exist

Just because I missed it, thanks for spotting it.
I'll send a new revision.